spec/features/guide_index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Guide index', type: :feature do
+  it "removes the Community part from guide community titles" do
+    guide = create(:guide)
+
+    visit root_path
+    rows = page.find(".guide-table").all("tr").map do |tr|
+      tr.all("td").map {|td| td.text}
+    end
+
+    expected_row = [
+      anything,
+      guide.latest_edition.content_owner.title.gsub(" Community", ""),
+      anything,
+      anything,
+      anything,
+    ]
+    expect(rows).to include expected_row
+  end
+end
```

Comment:
We have a helper method which might make for a nicer test. https://github.com/alphagov/service-manual-publisher/blob/master/spec/support/capybara_helpers.rb#L6-L8

Something like the following would be less susceptible to future changes to the table.

```
within_guide_index_row("Scrum") do
  expect(page).to have_css(".community", "Agile Delivery")
end
```

There are other options too obv..